reference/config_files/profiles.rst

```diff
@@ -286,7 +286,7 @@ the prefix of the variable. It is useful to automatically get the append/prepend
     MyPath1=+(path)/other path/path12
 
     # Unset the variable "MyPath1"
-    MyPath1=!
+    mypkg/*:MyPath1=!
```

Comment:
This would be introducing here the concept of per-package environment, but this section is to introduce the core syntax (add, append, remove).

It would be better to leave it as-is, and then add the per-package examples with ``mypkg/*`` separately

Reply:
I agree with you. There were two options, modifying this line or the below line. 
See this screen capture from https://docs.conan.io/2/reference/config_files/profiles.html#profiles

I was confused about reading `MyPath=!` and then the explanation below.


<img width="884" height="829" alt="Image" src="https://github.com/user-attachments/assets/5503a44d-27de-4a03-bc3c-0604e5b6c267" />TT